omod/src/main/webapp/pages/patient/encounterList.gsp

```diff
@@ -89,6 +89,7 @@ ${ ui.includeFragment("coreapps", "patientHeader", [ patient: patient.patient ])
 
         jq("#encounter-type-filter").change(function() {
            let typeName = jq(this).val();
+            console.log(jq(this).val());
```

Comment:
I assume this log statement was for debugging?  Do you want this committed?

Reply:
Bad!  lazy on my part, thanks, removing!